src/js/cf-tables.js

```diff
@@ -0,0 +1,164 @@
+
+/**
+ * cf-tables
+ * https://github.com/cfpb/cf-tables
+ *
+ * A public domain work of the Consumer Financial Protection Bureau
+ */
+
+( function( $ ) {
+  'use strict';
+
+  var SortableTable = function( table, options ) {
+    /*  At the moment, there are no default settings, but here's an object
+        for their future use! */
+    var defaults = {},
+    // settings is defaults combined with user options
+    settings = {},
+    // rows is an Array of Arrays, serving as a model of the table
+    rows = [],
+    $table = $( table ),
+    $headercells = $table.find( '.sortable' );
+
+    /**
+     * Initializes the SortableTable
+     * @param { object } options - Customizble options object
+     */
+    function _init( options ) {
+      settings = $.extend( {}, defaults, options );
+      _clickHandler();
+      // If the following classes exist, start by sorting those columns.
+      $table.find( '.sortable__start-up, .sortable__start-up' ).click();
+    }
+
+    /**
+     * Sorting function for Array.sort()
+     *
+     * @param { number } sign - A number where a negative number indicates a
+     * reverse sort.
+     * @param { sortType } sortType - A string used for sort types. By default,
+     * the values are sorted by their native type. If this value is set to
+     * 'number', then the cells' numeric values are used.
+     * @returns function - A function to be used by the Array.sort method, where
+     * the parameters 'a' and 'b' is each an Array (of Arrays) to be sorted
+     */
+
+    function _arraySorter( sign, sortType ) {
+      return function( a, b ) {
+        // Set a and b to the first Array in each Array-of-Arrays
+        a = a[0];
+        b = b[0];
+
+        // For number sort, convert a & b to numbers.
+        if ( sortType === 'number' ) {
+          a = Number( a.replace( /[^\d.-]/g, '' ) );
+          b = Number( b.replace( /[^\d.-]/g, '' ) );
+        }
+
+        // Sort the values
```

Comment:
Sure, but I left it verbose because I like for code like this to be easily readable. Passing a function to `Array.sort()` can be confusing.